source/server/BUILD

```diff
@@ -141,8 +141,8 @@ envoy_proto_library(
 
 envoy_cc_library(
     name = "hot_restarting_base",
-    srcs = envoy_select_hot_restart(["hot_restarting_base.cc"]),
-    hdrs = envoy_select_hot_restart(["hot_restarting_base.h"]),
+    srcs = ["hot_restarting_base.cc"],
+    hdrs = ["hot_restarting_base.h"],
```

Comment:
we probably don't need this nor the `#if !defined(__APPLE__)` block below, that was just me developing this in OS X... Once CI is happy can drop them.

Reply:
gone.